### DISCUSSION TITLE
Dont work method "get_clipboard()"

### DISCUSSION THREAD

**Discussion by vist-amur** [flet-dev/flet#837] [Q&A]
Hi! I try get clipboard (page.get_clipboard()), but get empty result.

**Comment by ndonkoHenri** [comment]
Can you show some code to be used for test?

**Comment by vist-amur** [comment]
yes of course!
```
for i in page.controls[0].controls:
    if str(type(i)).lower().find('text') > 0:
        if i.label == 'Barcode':
            i.value = page.get_clipboard()
page.update()
```

**Comment by vist-amur** [comment]
I understood what is problem! Need to request permission at browser, for do used clipboard and Its Ok. Now other question, how from the code to request this is permission?
